### DISCUSSION TITLE
Unable to set default Application User settings

### DISCUSSION THREAD

**Discussion by spkane** [gitpod-io/openvscode-server#199] [Q&A]
### Bug description

I am installing `openvscode-server` on an Ubuntu server and am trying to set some of the default User settings, so that all session have these set by default.

In particular I'd like to set these four:

```json
{
  "update.mode": "none",
  "telemetry.telemetryLevel": "off"
  "extensions.autoCheckUpdates": false,
  "extensions.autoUpdate": false
}
```

**NOTE**: These are not `remote` or `workspace` settings, but specifically `user` settings.

I can easily set them in a running session, but can not figure out a way to make these the default behavior for all sessions.

### Steps to reproduce

1) Try to create a file somewhere (e.g. `~/.openvscode-server/user-data/User/settings.json`) with the following contents.

```json
{
  "update.mode": "none",
  "telemetry.telemetryLevel": "off"
  "extensions.autoCheckUpdates": false,
  "extensions.autoUpdate": false
}
```

2) Start openvscode-server.

3) Open up and IDE session and check to see if these settings are set as defined in the settings file.

### Expected behavior

It should be possible to feed in a default set of Application User settings that will be used for all session, and can then be modified by the user for their active session.

**Comment by akosyakov** [comment]
VS Code Web does not store user settings on the disk, but in browser indexed DB per a origin. You will need to open it via User Settings and change there. After that they will be the same for any workspace which you open from such origin. If you change origin then settings won't be there.

**Reply by spkane** [reply]
@akosyakov Good to know. I thought something like that might be the case.

Given that, is there any way to change the default settings that browser session start with?

In this particular use case, I really want to ensure that these user settings are automatically set to these values when a new session is started, but unfortunately, it appears that they can only be set via user settings (2 of them can actually be set in a workspace as well). 

```
{
  "update.mode": "none",
  "telemetry.telemetryLevel": "off"
  "extensions.autoCheckUpdates": false,
  "extensions.autoUpdate": false
}
``

**Reply by akosyakov** [reply]
I guess the only way either build own fork and change defaults or preinstall the extension which does it. server.sh should accept `--install-extension` starting from 1.62, you can try it with nightly.

**Reply by spkane** [reply]
Thanks. I created a small extension to set these up on startup. https://github.com/spkane/vscode-training-tweaks

If I have any issues with that, then I might look at re-compiling the server, if I have to.
